components/places/src/history_sync/plan.rs

```diff
@@ -451,7 +454,7 @@ mod tests {
 
         // try and add an incoming record with the same URL but different guid.
         let record = HistoryRecord {
-            id: SyncGuid::new(),
+            id: SyncGuid::from(sync15::random_guid().unwrap()),
```

Comment:
After rc_crypto lands we'll probably have a better story for this, but that can be done in a follow up.